examples/sitepurpose.yml

```diff
@@ -0,0 +1,36 @@
+-
+  _valid: true
+  _name: 'grid-founding-member'
+  _data:
+    type: 'purchase'
+    label: 'Become a Founding Member'
+    cta: 'f6a4bd8d-55ec-4c3a-9ccd-9c5f3df80802'
```

Comment:
@bergie what is the cta uuid used for?
To edit a cta in the purpose array I should use the array index – not this id, right?